calico/felix/devices.py

```diff
@@ -141,8 +141,8 @@ def add_route(ip_type, ip, interface, mac):
     :param str mac: MAC address. May not be none unless ips is empty.
```

Comment:
This docstring line seems wrong (refers to `ips`, which doesn't exist), mind updating it?